-------------------------------------------------------------------------------
Test set: com.mayab.quality.integration.UserDAOTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.512 s <<< FAILURE! -- in com.mayab.quality.integration.UserDAOTest
com.mayab.quality.integration.UserDAOTest.testAddUserQry -- Time elapsed: 0.509 s <<< ERROR!
java.sql.SQLSyntaxErrorException: Unknown database 'calidad2024'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:825)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:446)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:239)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:188)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
	at org.dbunit.JdbcDatabaseTester.getConnection(JdbcDatabaseTester.java:116)
	at com.mayab.quality.integration.UserDAOTest.getConnection(UserDAOTest.java:49)
	at com.mayab.quality.integration.UserDAOTest.setUp(UserDAOTest.java:36)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

